### ISSUE TITLE
ps1 issue 3: are temp files being saved?

### ISSUE THREAD

**Issue by bturkus** [amiaopensource/ffmpeg-artschool#24]
i've run a few scripts, and it looks like temp files for ffplay aren't being deleted? is tis expected and/or problematic?

```
Directory of C:\Users\bt133523\AppData\Local\Temp

11/10/2020  11:41 AM    <DIR>          .
11/10/2020  11:41 AM    <DIR>          ..
10/26/2020  10:34 AM    <DIR>          chocolatey
11/10/2020  11:33 AM        67,113,541 tmp3732.tmp
11/10/2020  11:41 AM        65,628,777 tmpE53F.tmp
               2 File(s)    132,742,318 bytes
               3 Dir(s)  202,294,116,352 bytes free
```

**Comment by nkrabben** [comment]
Oh cool
https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettempfilenamea?redirectedfrom=MSDN

Looks like I have to delete them myself. Will do

**Comment by nkrabben** [comment]
fixed in  0d7fd83
